Tests/SentryTests/Transactions/SentrySpanKeyTests.swift

```diff
@@ -0,0 +1,12 @@
+@testable import Sentry
+import XCTest
+
+class SentrySpanKeyTests: XCTestCase {
+    func testFileSize_shouldBeExpectedValue() {
```

Comment:
💯 for the tests so we have to change up the constant values in two places to avoid changing it by accident.